src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -365,6 +365,9 @@
         - name: --disable-disk-driver
           type: bool
           short-summary: Disable AzureDisk CSI Driver.
+        - name: --azuredisk-csi-version
```

Comment:
it should be `disk_version`, there is already a draft PR: https://github.com/Azure/azure-cli-extensions/pull/4772, while since the rollout is not ready on canary region, we cannot test now.

Reply:
As discussed, we decided to use ```disk_driver_version```